doc/.wordlist.txt

```diff
@@ -37,3 +37,7 @@ Jira
 VM
 YAML
 CephFS
+disaggregated
+subnets
```

Comment:
@ru-fu I'm not sure why these need to be added to the wordlist. Where does sphinx/spellcheck get it's default wordlist from?

Reply:
It's using aspell. Not sure why it doesn't know "disaggregated" (it's not the most common word though ;) ). Unfortunately, aspell doesn't understand plurals for added words, that's why "subnets" must be added manually.

That being said, these additions should all go into `.custom_wordlist.txt` though instead of `.wordlist.txt` @gabrielmougard .

Reply:
Ah fair enough, thanks :)

Reply:
fixed